gratipay/models/team.py

```diff
@@ -54,3 +54,29 @@ def create_new(cls, owner, fields):
     def update_receiving(self, cursor=None):
         # Stubbed out for now. Migrate this over from Participant.
         pass
+
+
+    def migrate_tips(self):
+        subscriptions = self.db.all("SELECT * FROM subscriptions WHERE team=%s", (self.slug,))
+
+        # Make sure the migration hasn't been done already
+        if subscriptions:
+            raise AlreadyMigrated
+
+        self.db.run("""
+
+            INSERT INTO subscriptions
+                        (ctime, mtime, subscriber, team, amount, is_funded)
+                 SELECT ctime
+                      , mtime
```

Comment:
Hmm, I'm not sure. We use `mtime` to show the time the tip was last modified in the UI (Last set). Do we want that to be the time when the user last set it, or the time when we migrated the tip?